superbench/benchmarks/micro_benchmarks/gpu_copy_performance/gpu_copy.cu

```diff
@@ -517,6 +559,19 @@ int RunCopy(BenchArgs *args) {
                 cudaMemcpyAsync(sub.dst_dev_gpu_buf_ptr, sub.src_dev_gpu_buf_ptr, args->size, cudaMemcpyDefault,
                                 sub.stream);
             }
+            if (i + 1 == num_warm_up) {
+                cuda_err = cudaEventRecord(sub.start_event, sub.stream);
+                if (cuda_err != cudaSuccess) {
+                    fprintf(stderr, "RunCopy::cudaEventRecord error: %d\n", cuda_err);
```

Comment:
do you want to use an macro for this?
and if you check error for cuda event, why not check it for copy kernel or cudaMemcpy?

Reply:
Not very efficient to do customized printing with macro? Add check for cudaMemcpy. For kernels, the error code will be returned in cudaStreamSynchronize.

Reply:
> Not very efficient to do customized printing with macro? 

[for example](https://github.com/NVIDIA/nccl-tests/blob/7130fa6096466f80b0c310b9a3070b6556c0e158/src/common.h#L25-L27)

Reply:
Great, maybe next PR.